Framework/Algorithms/src/ReflectometryReductionOneAuto2.cpp

```diff
@@ -1000,6 +1004,40 @@ bool ReflectometryReductionOneAuto2::processGroups() {
   return true;
 }
 
+/** Get the output workspace names for a workspace in a group.
+ * If an input workspace has been passed with the format
+ * TOF_<runNumber>_<otherInfo> then the output workspaces will be of the same
+ * format otherwise they are numbered according to the wsGroupNumber
+ */
+ReflectometryReductionOneAuto2::WorkspaceNames
+ReflectometryReductionOneAuto2::getOutputNamesForGroups(std::string inputName,
+                                                        std::string runNumber,
+                                                        size_t wsGroupNumber) {
+  auto const output = getOutputWorkspaceNames();
+  std::string informativeName = "TOF" + runNumber + "_";
+  bool informativeInput = false;
+  if (equal(informativeName.begin(), informativeName.end(),
+            inputName.begin())) {
+    informativeInput = true;
+  }
+
+  WorkspaceNames outputNames;
+  if (informativeInput) {
+    auto informativeTest = inputName.substr(informativeName.length());
```

Comment:
I don't think this needs to be two separate if statements.
We can move the code within the `if (informativeInput)` block to the `if (equal....)` block